packages/block-library/src/spacer/edit.js

```diff
@@ -123,6 +123,7 @@ const SpacerEdit = ( {
 			return (
 				<ResizableSpacer
 					minWidth={ MIN_SPACER_SIZE }
+					maxWidth={ MAX_SPACER_SIZE }
```

Comment:
With this change, it won't be possible to drag the resizable box beyond the max allowed size.

Previously, the user was allowed to drag the box larger than it could be, and then the box would suddenly snap back to its maximum allowed size when the user stopped dragging.

Reply:
This is not true anymore after this [commit](https://github.com/WordPress/gutenberg/pull/39577/commits/15a81af4fedcadeccee3072c6ac0f9f2ab791788), marking as resolved.